manager/src/main/resources/k8s/templates/streamsApp/deployment.th.yaml

```
@@ -48,3 +48,5 @@ spec:
             requests:
               memory: [(${resourceConfig.memory.request})]
               cpu: [(${resourceConfig.cpu.request})]
+      imagePullSecrets: [# th:if="${imagePullSecret}"]
```

I think I'd prefer something like this:
```suggestion
      [# th:if="${imagePullSecret}"]
      imagePullSecrets: 
```

---

If we make it this way, then secret=null leads to an empty ArrayList in the field imagePullSecrets.
In the original version, we have imagePullSecrets=null.
Do we really want to have an empty array here or null?

---

Empty list seems to be the better choice then I think